pkg/config/tlsconfig/config.go

```diff
@@ -107,6 +108,9 @@ type FileReader interface {
 // Client is client-side configuration for server TLS.
 type Client struct {
 	FileReader         FileReader `json:"-" yaml:"-" name:"-"`
+	loadRootCA         sync.Once  `json:"-" yaml:"-" name:"-"`
+	rootCABytes        []byte     `json:"-" yaml:"-" name:"-"`
+	rootCABytesError   error      `json:"-" yaml:"-" name:"-"`
```

Comment:
I'm quite sure that unexported fields are already ignored by json, yaml and the config package.